1.0/docs/devguide/properties.md

```diff
@@ -370,40 +370,59 @@ Example:
 In addition to properties, observers can also observe [paths to sub-properties](#observing-path-changes),
 [paths with wildcards](#deep-observation), or [array changes](#array-observation).
 
-### Observing path changes {#observing-path-changes}
+### Observing sub-property changes {#observing-path-changes}
 
-You can also observe changes to object sub-properties using the 
-`observers` array, by specifying a full path (`user.manager.name`)
-as a function argument.
+To observe changes in object sub-properties:
 
-Example:
-
-    Polymer({
+1.  Initialize your object to a default value.
+2.  Define an `observers` array.
+3.  For each object sub-property that you want to observe, add an element to
+    the `observers` array. Each element must be a method call that 
+    accepts a single argument. The argument is the path to the object
```

Comment:
method call -> this is difficult one. What you're really specifying is a method _name_, followed by the path you want to observe in parenthesis. I wouldn't emphasize the "single argument" angle -- the observers array can be used to specify path observers, deep path observers, and multiple-property (or path) observers, like;

    observers: [
      'observeSomeStuff(cats,dogs,primates.*,dolphins.bottlenose)'
    ]

In the existing docs, we've used the term "dependent property" (or path would be more accurate) to refer to the stuff inside an observer or computed property definition. The distinction is between the thing being observed (the dependent property/path) and the value sent to the observer method (the argument).